client/components/LearnerViewDirective.js

```diff
@@ -492,10 +492,15 @@ tie.directive('learnerView', [function() {
             feedbackDiv.scrollTop = feedbackDiv.scrollHeight +
               additionalHeightForLoadingIndicator;
             $timeout(function() {
+              // Tasks from the first to current.
+              var tasksToCurrent = [];
```

Comment:
tasksToCurrent is a bit confusing. Maybe orderedTasks or something? What do you think?